package.json

```diff
@@ -4,8 +4,8 @@
   "version": "2.1.0",
   "author": {
     "name": "Samuel Gratzl",
-    "email": "samuel.gratzl@datavisyn.io",
-    "url": "http://datavisyn.io"
+    "email": "sam@sgratzl.com",
```

Comment:
```suggestion
    "email": "contact@datavisyn.io",
```

Reply:
When pushed back to the datavisyn repo, there should be the datavisyn credentials